### ISSUE TITLE
Path-layer has visual artifacts when rendering small segments (new coord system)

### ISSUE THREAD

**Issue by georgios-uber** [uber/deck.gl#2384]
Path-layer has visual artifacts when rendering small segments using new coord system.
Especially pronounced when rendering curves using [nebula curve utils](https://github.com/uber/nebula.gl/blob/master/modules/core/src/lib/curve-utils.js)

#### Actual Result
<img width="143" alt="screen shot 2018-10-24 at 5 46 26 pm" src="https://user-images.githubusercontent.com/25012840/47469242-c391d680-d7b4-11e8-8bdf-a851e6374cab.png">

#### Expected Result
Smooth line.

#### To Do List 
  - [ ] Add label and assign to milestone
  - [ ] Coding
  - [ ] Test

**Comment by 1chandu** [comment]
To be clear, this issue doesn't occur when using `COORDINATE_SYSTEM.LNGLAT_DEPRECATED`?

**Comment by georgios-uber** [comment]
Yes, but is not significant enough to block or revert any releases.

**Comment by 1chandu** [comment]
Could be related to path-layer shader hardcoding segment size.

**Comment by georgios-uber** [comment]
I might be able to work on this today. Any suggestions where to start?
Where is the hardcoded size?
